<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1297</width>
    <height>849</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="setting_groupbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>271</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QLabel" name="date_label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Prediction</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="predict_lineedit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="config_pushbutton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>251</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Setting</string>
     </property>
    </widget>
    <widget class="QLabel" name="model_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Prediction Model</string>
     </property>
    </widget>
    <widget class="QComboBox" name="model_combobox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>LSTM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GRU</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NP-LSTM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NP-GRU</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="class_combobox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Soft Ground Depth</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Clay Ratio</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="class_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Class Method</string>
     </property>
    </widget>
    <widget class="QComboBox" name="mode_combobox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Prediction</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Test</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="mode_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Prediction Mode</string>
     </property>
    </widget>
    <widget class="QLabel" name="present_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Current</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="present_lineedit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="update_checkbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Model Update</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="constrain_checkbox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Constrain Last</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="file_groupbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QLabel" name="filename_label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>21</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="text">
      <string>file name...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openfile_pushbutton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="text">
      <string>Open File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="datasetting_pushbutton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>251</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="text">
      <string>Detailed Data Settings</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="log_groupbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>276</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Log</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTextBrowser" name="log_textbrowser"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="export_groupbox">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>630</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Export Setting</string>
    </property>
    <widget class="QPushButton" name="img_pushbutton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Export Figure</string>
     </property>
    </widget>
    <widget class="QPushButton" name="csv_pushbutton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Export .csv</string>
     </property>
    </widget>
    <widget class="QLabel" name="figure_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Graph as Image File</string>
     </property>
    </widget>
    <widget class="QLabel" name="csv_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Save as CSV File</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="prediction_pushbutton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>271</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Settlement Prediction</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressbar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="graph_groupbox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>781</width>
      <height>791</height>
     </rect>
    </property>
    <property name="title">
     <string>Graph</string>
    </property>
    <widget class="QLabel" name="height_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Fill Height Setting</string>
     </property>
    </widget>
    <widget class="PlotWidget" name="height_graph" native="true">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>50</y>
       <width>751</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="height_pushbutton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Setting</string>
     </property>
    </widget>
    <widget class="PlotWidget" name="settlement_graph" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>751</width>
       <height>431</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="help_pushbutton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>20</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>How to Use?</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="status_groupbox">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>10</y>
      <width>201</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Status</string>
    </property>
    <widget class="QLabel" name="height_status_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Soft Ground Depth : (m)</string>
     </property>
    </widget>
    <widget class="QLabel" name="clay_status_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Clay : (m)</string>
     </property>
    </widget>
    <widget class="QLabel" name="class_status_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Class Method : </string>
     </property>
    </widget>
    <widget class="QLabel" name="model_status_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Prediction Model : </string>
     </property>
    </widget>
    <widget class="QLabel" name="present_date_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Observation Days : </string>
     </property>
    </widget>
    <widget class="QLabel" name="predict_date_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Prediction Days :</string>
     </property>
    </widget>
    <widget class="QLabel" name="silt_status_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Silt : (m)</string>
     </property>
    </widget>
    <widget class="QLabel" name="sand_status_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Sand : (m)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="prediction_groupbox">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>429</y>
      <width>201</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Prediction Value</string>
    </property>
    <widget class="QLabel" name="present_value_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Settlement : (cm)</string>
     </property>
    </widget>
    <widget class="QLabel" name="pred_value_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Final Pred Settlement : (cm)</string>
     </property>
    </widget>
    <widget class="QLabel" name="real_value_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Final Act Settlement : (cm)</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_gap_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Final Settlement Gap : (cm)</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1297</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
